packages/js/product-editor/src/components/content-preview/content-preview.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import { createElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { sanitizeHTML } from '../../utils/sanitize-html';
+
+type ContentPreviewProps = {
+	content: string;
+};
+
+const CONTENT_TAGS = [
+	'a',
+	'b',
+	'em',
+	'i',
+	'strong',
+	'p',
+	'br',
+	'img',
+	'blockquote',
+	'cite',
+	'h1',
+	'h2',
+	'h3',
+	'h4',
+	'h5',
+	'h6',
+	'ul',
+	'li',
+	'ol',
+	'div',
+];
```

Comment:
Nice!

Reply:
Thanks for the review, Lourens!

>It looks like when I type some text and hit back some of the changes are not being reflected, I assume this is because of the useDebounce, although this will probably be fixed if we make use of the buttons within the Modal.

I notice this even without debounce.  I'll create an issue to look into this.  I believe there may already be some level of debouncing occurring within the Gutenberg components.

>There appears to be some missing styling still, for example when I add a button, the initial button styling is missing and I can't justify it to the right or to the left. This is reflected correctly on the front-end. The alignment issue mostly pertains to missing css that sets the block to flex.

Good catch!  I'll create an issue around this as well.

I'm going to merge this one as-is since two issues are unrelated to this PR and I'll get some issues created to tackle both of these in follow-ups.